airbrake.go

```diff
@@ -10,13 +10,21 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// default levels to be fired when logging on
+var defaultLevels = []logrus.Level{
+	logrus.ErrorLevel,
+	logrus.FatalLevel,
+	logrus.PanicLevel,
+}
+
 // AirbrakeHook to send exceptions to an exception-tracking service compatible
 // with the Airbrake API.
 type airbrakeHook struct {
 	Airbrake *gobrake.Notifier
+	levels   []logrus.Level
 }
 
-func NewHook(projectID int64, apiKey, env string) *airbrakeHook {
+func NewHook(projectID int64, apiKey, env string, levels ...logrus.Level) *airbrakeHook {
```

Comment:
If `levels` was exported (see my comment above), we could keep the same API.
Users would have to use `hook.Levels = []logrus.Level{logrus.ErrorLevel,
		logrus.FatalLevel}` for example.

Reply:
no problem